Cargo.toml

```diff
@@ -14,6 +14,9 @@ license.workspace = true
 edition.workspace = true
 repository.workspace = true
 
+[features]
+time = ["dep:time"]
```

Comment:
What do you think of enabling this by default, as time is such a small crate (I guess)?

Reply:
I made it opt-in because currently we need the interop only for one operator (secret-operator).

Reply:
I know ;) Was just an idea to not need users to know this is behind a feature flag. But I don't care, was just an idea :)